### PR TITLE
chore: add defaults to `just test-compatibility`

### PR DESCRIPTION
**Pull request by dpc**

### CODE REVIEW

**Review by bradleystachurski — APPROVED** [review]

**Review by m1sterc001guy — APPROVED** [review]

File: justfile.fedimint.just [hunk 1]
```diff
@@ -22,10 +22,10 @@ test-ci-all:
 test-count:
   ./scripts/tests/test-cov.sh
 
-test-compatibility *VERSIONS:
+test-compatibility *VERSIONS="v0.2.1":
```
Comment (elsirion) [inline justfile.fedimint.just:25]: Is the idea here that this is our MSFMV for now?

Comment (dpc) [reply justfile.fedimint.just:25]: I think so, but pinging @bradleystachurski .

Comment (bradleystachurski) [reply justfile.fedimint.just:25]: The motivation for adding the default was to make it less confusing for folks when calling the just command. There was confusion reported in [Discord](https://discord.com/channels/990354215060795454/1217141774812713008/1217253961631924274).

Does MSFMV == Minimum Supported Fedimint Version?

Comment (dpc) [reply justfile.fedimint.just:25]: > Does MSFMV == Minimum Supported Fedimint Version?

Certainly. :)

**Pull request merged**